README.md

```diff
@@ -1,92 +1,68 @@
 # ROS2 Migration Tools
 
-This package contains tool to simplify migrating a package from ROS1 to ROS2.
+This package contains tool to simplify migrating a package from ROS1 to ROS2. This package contains 3 tools: `docker_setup` for launching a docker container that has ROS2 and ROS1 installed, `check_deps` to check which dependencies have already been migrated to ROS2, and `porting_tools` which contain scripts to make some automatic changes to source code and build files and warn about manual changes.
 
-How to build:
+Note: Any of the tools can be run with the `--help` option for the full list of options.
 
-In the root directory run pip3 install -e .
+## Setup
+### Docker
+Install docker https://docs.docker.com/install/
 
-How to run:
+### Folder Structure
+ 1. Create a new directory : `mkdir ros_migration_ws/src -p`
+ 2. Initialize a Catkin workspace: `cd ros_migration_ws/src ; catkin_init_workspace`
```

Comment:
Did you find catkin_init_workspace to be necessary? We don't actually build the ros1 package and shouldn't need the root level CMakeLists.txt that it creates.